platform/default/src/mbgl/storage/offline_download.cpp

```diff
@@ -469,20 +469,30 @@ void OfflineDownload::ensureResource(Resource&& resource, std::function<void(Res
     auto workRequestsIt = requests.insert(requests.begin(), nullptr);
     *workRequestsIt = util::RunLoop::Get()->invokeCancellable([=, this]() {
         requests.erase(workRequestsIt);
-        const auto resourceKind = resource.kind;
+        auto res = resource;
```

Comment:
Nit: `res` can be `response` or `resource` so maybe it is better not to shorten it.